test/jasmine/tests/isosurface_test.js

```diff
@@ -313,7 +313,7 @@ describe('Test isosurface', function() {
         });
     });
 
-    describe('@noCI hover', function() {
+    describe('hover', function() {
```

Comment:
Not anything new, but I noticed this test

https://github.com/plotly/plotly.js/blob/e938c407c5889b60d04ee9cbdc3835bfba111a02/test/jasmine/tests/isosurface_test.js#L272

is missing a `@gl` tag.